server/lib/src/entry.rs

```diff
@@ -354,35 +354,42 @@ impl Entry<EntryInit, EntryNew> {
                 } else {
                 let attr = AttrString::from(k.to_lowercase());
                 let vv: ValueSet = match attr.as_str() {
-                    "attributename" | "classname" | "domain" => {
+                    kanidm_proto::constants::ATTR_ATTRIBUTENAME | kanidm_proto::constants::ATTR_CLASSNAME | kanidm_proto::constants::ATTR_DOMAIN => {
```

Comment:
hey, I want to remove this function too, this one is awful.

Reply:
me2me2